### PR TITLE
:ambulance: error with pos.credit.update records with not debt journal

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

**Review by ilmir-k — APPROVED** [review]

**Comment by ilmir-k** [comment]
@yelizariev

File: pos_debt_notebook/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,11 @@
+`5.3.2`
+-------
+
+- **FIX:** Error in getting customer history when a journal was unset as ``Debt`` one. Or when manual credit update was created with not debt journal
+- **FIX:** "Could not get property 'debt' of undefined" Error after creating a new customer in POS
+- **FIX:** Not working writing-off wizard with 'New Balance' option
+- **Improvement:** Always refreshing paymentlines on paymentscreen opening
```
Comment (yelizariev) [inline pos_debt_notebook/doc/changelog.rst:7]: Да, пришлось вам поработать в Панаме :+1:

**Review by yelizariev — APPROVED** [review]
LGTM

**Comment by yelizariev** [comment]
@KolushovAlexandr please fix lint:


pos_debt_notebook/models.py:801: [C8107(translation-required), PosCreditUpdate._check_journal_id] String parameter on "ValidationError" requires translation. Use _('Credit Updates may be created only for credit journals !')

**Comment by yelizariev** [comment]
Also, run

```
pre-commit run --all
```

**Comment by yelizariev** [comment]
> @KolushovAlexandr please fix lint:
> 
> pos_debt_notebook/models.py:801: [C8107(translation-required), PosCreditUpdate._check_journal_id] String parameter on "ValidationError" requires translation. Use _('Credit Updates may be created only for credit journals !')

Ok, I'll do it myself

**Comment by yelizariev** [comment]
I approve to merge it now

**Pull request merged**